sklearn/utils/extmath.py

```diff
@@ -349,6 +349,13 @@ def randomized_svd(M, n_components, n_oversamples=10, n_iter=4,
     n_random = n_components + n_oversamples
     n_samples, n_features = M.shape
 
+    if n_iter == None:
```

Comment:
`if n_iter is None`

Even if that doesn't make such a big difference, this is the usual way of comparing against None.